src/context/verification.context.tsx

```diff
@@ -71,7 +70,7 @@ export const VerificationProvider = ({ children }: { children: ReactNode }) => {
 						break;
 
 					default:
-						showToastError(error);
+						console.log('getVerificationData error: ', error);
```

Comment:
If we remove showToast then how are we informing user? Is this approved behaviour by the product owners?

Reply:
I've replaced the error component. Now I'll show the error in the verification page body instead of a toast on top the page we don't need a top toast error anymore.
![image](https://github.com/Giveth/giveth-dapps-v2/assets/73441857/a082984b-b821-4de9-8900-05da877591f7)